src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -38,6 +38,13 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <NETSdkError Condition="'$(SelfContained)' == 'true'"
                  ResourceName="PackAsToolCannotSupportSelfContained" />
+    
+    <NETSdkError Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'"
+                 ResourceName="DotnetToolOnlySupportNetcoreapp" />
+    
+    <!-- Compare version in System.Version type, if it is lower than 2.1 -->
+    <NETSdkError Condition='$([System.Version]::Parse("$(_TargetFrameworkVersionWithoutV)").CompareTo($([System.Version]::Parse("2.1")))) &lt; 0'
```

Comment:
@nguerrera as we discussed. use msbuild compare may cause problem on 2.11 vs 2.2. This looks ugly but it works. @rainersigwald is it supported? I cannot find a direct document about it.

Also $([System.Version]::Parse("2.1")) seems keep the type. If I just replace it with string 2.1 it will error and ask for Version type.

The alternative is to write a Task. It is a lot of code and also it might not be cleaner than that.

Reply:
Reverted to the original